test/fields/test_files.py

```diff
@@ -238,6 +238,9 @@ def test_bad_inputdir():
         ArgSchemaParser(input_data=input_data,
                         schema_type=BasicInputDir, args=[])
 
+@pytest.mark.skipif(sys.platform == "win32",
+                    reason="can't get working after migrating from appveyor to"
+                           "github-actions.")
 def test_inputdir_no_access(tmpdir):
```

Comment:
this is definitely a long term sticking point and we should figure out a way to test it.... not being able to read a file or write a file is one of the main features of using this, and our implementations have definitely stopped working on windows at times and slowed work.